pom.xml

```
@@ -263,7 +263,7 @@
                             <goal>generate</goal>
                         </goals>
                         <configuration>
-                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/domains/ecommerce-app/api/ecommerce-payment-methods-service/v1/_openapi.json.tpl</inputSpec>
+                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/CHK-1556-added-jwt-policy-fees/src/domains/ecommerce-app/api/ecommerce-payment-methods-service/v1/_openapi.json.tpl</inputSpec>
```

```suggestion
                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/domains/ecommerce-app/api/ecommerce-payment-methods-service/v1/_openapi.json.tpl</inputSpec>
```

Remember to restore this ref to main branch after infra branch merge